src/stubs/controller.model.stub

```diff
@@ -21,21 +21,16 @@ class {{ class }} extends Controller
 
     /**
      * Show the form for creating a new resource.
-     *
-     * @return \Illuminate\Http\Response
      */
-    public function create()
+    public function create(): View
     {
         return view('{{ resourcePluralVariable }}.create');
     }
 
     /**
      * Store a newly created resource in storage.
-     *
-     * @param  \Illuminate\Http\Request $request
-     * @return \Illuminate\Http\Response
      */
-    public function store(Request $request)
+    public function store(Request $request): Response
```

Comment:
I did a quick manual test on this and I've got the following error:
```
Return value must be of type Illuminate\\Http\\Response, Illuminate\\Http\\RedirectResponse returned
```
I can't put more time on this at the moment. Have you tested it ?